pages/events/index.tsx

```diff
@@ -1,12 +1,86 @@
 import { Typography } from 'components/atoms/Typography/Typography';
+import { Button } from 'components/atoms/Button/Button';
 import Link from 'next/link';
+import Image from 'next/image';
+import UserIcon from 'components/icons/UserIcon.svg';
+import Ticket from 'components/icons/Ticket.svg';
+import { useGetAllEventsQuery } from 'generated/graphql';
 
 const EventsPage = () => {
+	const { data, loading } = useGetAllEventsQuery();
+	console.log(data, loading);
```

Comment:
as far as I know we have eslint rule to warn you about console logs, if you dont see that check for config

Reply:
Resovled in next commit